gems/sorbet-runtime/lib/types/private/types/void.rb

```diff
@@ -6,17 +6,43 @@
 class T::Private::Types::Void < T::Types::Base
   ERROR_MESSAGE = "Validation is being done on an `Void`. Please report this bug at https://github.com/sorbet/sorbet/issues"
 
-  # The actual return value of `.void` methods.
-  #
-  # Uses `module VOID` because this gives it a readable name when someone
-  # examines it in Pry or with `#inspect` like:
-  #
-  #     T::Private::Types::Void::VOID
-  #
-  module VOID
+  class VoidSingleton < BasicObject
+    def inspect
+      "T::Private::Types::Void::VOID"
+    end
+
+    ALLOWED_KERNEL_METHODS = [:object_id, :class, :respond_to?, :eql?]
+
+    if defined?(::PP::ObjectMixin)
```

Comment:
Is it appropriate to include these kind of helpers?

They don't perfect fit here, but checking `if defined?(Sorbet)` in those projects would be equally strange.

Without these, `pp` or `ap` on a void method just raises.